ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/ldes/retentionpolicy/creation/VersionBasedRetentionPolicyCreator.java

```diff
@@ -1,12 +1,39 @@
 package be.vlaanderen.informatievlaanderen.ldes.server.domain.ldes.retentionpolicy.creation;
 
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.converter.RdfModelConverter;
 import be.vlaanderen.informatievlaanderen.ldes.server.domain.ldes.retentionpolicy.RetentionPolicy;
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.ldes.retentionpolicy.versionbased.VersionBasedRetentionPolicy;
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.tree.member.repository.MemberRepository;
 import org.apache.jena.rdf.model.Model;
+import org.apache.jena.rdf.model.Property;
+import org.apache.jena.rdf.model.RDFNode;
+import org.apache.jena.rdf.model.Statement;
+import org.apache.jena.riot.Lang;
+
+import java.util.List;
+
+import static be.vlaanderen.informatievlaanderen.ldes.server.domain.constants.RdfConstants.LDES;
+import static org.apache.jena.rdf.model.ResourceFactory.createProperty;
 
 public class VersionBasedRetentionPolicyCreator implements RetentionPolicyCreator {
+	public static final Property LDES_AMOUNT = createProperty(LDES, "amount");
+	private final MemberRepository memberRepository;
+
+	public VersionBasedRetentionPolicyCreator(MemberRepository memberRepository) {
+		this.memberRepository = memberRepository;
+	}
+
 	@Override
 	public RetentionPolicy createRetentionPolicy(Model model) {
-		// TODO VSDSPUB-713
-		return null;
+		List<Statement> ldesAmountStatements = model.listStatements(null, LDES_AMOUNT, (RDFNode) null).toList();
```

Comment:
Als je enkel interesse hebt in de objects, vind ik het properder om die dan ook gewoon direct op te halen ipv de volledige statements:

`		List<RDFNode> ldesAmountStatements = model.listObjectsOfProperty(LDES_AMOUNT).toList();
`